### PR TITLE
Adding User properties and Transitive search to v1.0

### PR DESCRIPTION
**Pull request by anchandaAADGraph**
Adding User properties
Enabling Transitive member search

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c957f9e](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/c957f9ec8507f35fabf49d453f6aadbdacbead5d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c957f9e](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/c957f9ec8507f35fabf49d453f6aadbdacbead5d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c957f9e](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/c957f9ec8507f35fabf49d453f6aadbdacbead5d)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 56 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c957f9e](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/c957f9ec8507f35fabf49d453f6aadbdacbead5d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/device-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/device-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/group-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list-transitivemembers.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/group-list-transitivemembers.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/user-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/user-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/group.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/group.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/user.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/user.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C30%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901300011461531-3888%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cWs%2FIzUJ5aBAxx5pAudRdEj7igjkudwv50RZTRXKLSY%3D&st=2019-01-30T00%3A10%3A18Z&se=2019-03-02T00%3A15%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9888431](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/988843153630c68863e3757aab86332accfc2e15)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9888431](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/988843153630c68863e3757aab86332accfc2e15)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

File: api-reference/v1.0/api/group-list-transitivemembers.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  "isCollection": true
+} -->
+
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+
+{
+  "value": [
+    "@odata.type": "#microsoft.graph.user",
```
[the remark this answers is not in the document]
Comment (anchandaAADGraph) [reply api-reference/v1.0/api/group-list-transitivemembers.md]: Addressed.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9888431](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/988843153630c68863e3757aab86332accfc2e15)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 56 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9888431](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/988843153630c68863e3757aab86332accfc2e15)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/device-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/device-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/group-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list-transitivemembers.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/group-list-transitivemembers.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/user-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/user-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/group.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/group.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/user.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/user.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C30%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901301919108236-3888%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=V09ACOPAWi%2BwyZEKEx0rgOdhX1M9Vs3zg7xywFAxHc8%3D&st=2019-01-30T19%3A17%3A49Z&se=2019-03-02T19%3A22%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[7ba713b](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/7ba713b5186a94f11142101fbc4fdc47024c7956)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[7ba713b](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/7ba713b5186a94f11142101fbc4fdc47024c7956)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[7ba713b](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/7ba713b5186a94f11142101fbc4fdc47024c7956)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 56 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[7ba713b](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/7ba713b5186a94f11142101fbc4fdc47024c7956)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/device-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/device-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/group-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list-transitivemembers.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/group-list-transitivemembers.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/user-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/user-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/group.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/group.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/user.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/user.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C30%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901302202418780-3888%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=O9maKt8mZWWSd6tmUfWXWU1lMC4Gs3gXf%2BkPILpWGwI%3D&st=2019-01-30T22%3A01%3A11Z&se=2019-03-02T22%3A06%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a590b6b](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/a590b6b1cfbb015968db588415c3a0fd7e41f088)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a590b6b](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/a590b6b1cfbb015968db588415c3a0fd7e41f088)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a590b6b](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/a590b6b1cfbb015968db588415c3a0fd7e41f088)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 56 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a590b6b](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/a590b6b1cfbb015968db588415c3a0fd7e41f088)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/changelog.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/concepts/changelog.md) | :warning:Warning | | [Details](#user-content-concepts%2Fchangelog.md)
[api-reference/v1.0/api/device-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/device-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/group-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list-transitivemembers.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/group-list-transitivemembers.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/user-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/user-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/group.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/group.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/user.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/user.md) | :white_check_mark:Succeeded | |

<a name="concepts%2Fchangelog.md"></a>
### [concepts/changelog.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/concepts/changelog.md)
 - **Line 33**: **[Warning]** Invalid file link:(~/../api-reference/v1.0/resources/user.md).
 - **Line 35**: **[Warning]** Invalid file link:(~/../api-reference/v1.0/resources/group.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C30%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901302235344779-3888%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=QbnQVvsTLRY%2FSwXiOSNQJV1U6%2BS7oQgNnpi4GrSWk%2BY%3D&st=2019-01-30T22%3A35%3A24Z&se=2019-03-02T22%3A40%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by dkershaw10** [comment]
@Lauragra https://github.com/microsoftgraph/microsoft-graph-docs/pull/3741 has the changelog updates.  None of these changes update the TOC.  This is good to go to merge together with #3741.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ec20f3c](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/ec20f3c91875eba7e8d55701264e9d009a9cbef0)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ec20f3c](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/ec20f3c91875eba7e8d55701264e9d009a9cbef0)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ec20f3c](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/ec20f3c91875eba7e8d55701264e9d009a9cbef0)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 57 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ec20f3c](https://github.com/anchandaAADGraph/microsoft-graph-docs/pull/3888/commits/ec20f3c91875eba7e8d55701264e9d009a9cbef0)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/changelog.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/concepts/changelog.md) | :warning:Warning | | [Details](#user-content-concepts%2Fchangelog.md)
[api-reference/v1.0/api/device-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/device-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/group-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list-transitivemembers.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/group-list-transitivemembers.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/user-list-transitivememberof.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/api/user-list-transitivememberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/group.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/group.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/user.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/api-reference/v1.0/resources/user.md) | :white_check_mark:Succeeded | |

<a name="concepts%2Fchangelog.md"></a>
### [concepts/changelog.md](https://github.com/anchandaAADGraph/microsoft-graph-docs/blob/user_transitive_v1mapping3/concepts/changelog.md)
 - **Line 37**: **[Warning]** Invalid file link:(~/../api-reference/v1.0/resources/user.md).
 - **Line 39**: **[Warning]** Invalid file link:(~/../api-reference/v1.0/resources/group.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C31%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901312048187778-3888%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=AtV%2Bz9QkgECaDI%2FNsontB9Y7OTzUGiRJfwAwPDHI%2Bbc%3D&st=2019-01-31T20%3A47%3A54Z&se=2019-03-03T20%3A52%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**